cmd/csi-resizer/main.go

```diff
@@ -99,7 +102,7 @@ func main() {
 
 	csiResizer, err := resizer.NewResizer(
 		*csiAddress,
-		*csiTimeout,
+		*timeout,
```

Comment:
this means that if someone was using deprecated parameter - `csiTimeout`, it will simply be ignored. Can you fix it?

Reply:
Since in this release we are only deprecating the older `csiTimeout` parameter (as evident from CLI help flags), but if we remove the flag in the same release we will have problems.

Reply:
alternately - if we are not going to support `csiTimeout` flag anymore, we should remove it alogether. But I am in favour of deprecating the flag for now and removing it later. cc @msau42

Reply:
If someone sets csiTimeout, can we have it set timeout?

I'm also ok with just removing the flag because we're doing a major release anyway.

Reply:
Ok.. let me try to explain what I was thinking. Being this is a major release, we could definitely mark `csitimeout` param as a "deprecated" one , also we could mention in the release note that if someone set this value , it will be NO-OP and the default timeout value in the external resizer will be used. The advantage with this approach is that, the older templates ( which set `csiTimeout` ) in some customer/user setup could also work , but its a NO-OP . If they set `timeout` while using new templates in parity with this release, it will be effective as well. Isnt it a good idea to address this way? @gnufied @msau42 wdyt?

Reply:
that would be problematic. It would seem like  we still support `csiTimeout` option but in truth we don't. If we are going to not support `csiTimeout` option then we should just remove it.

Reply:
@gnufied @msau42 ok.. are we in agreement to completely remove `csiTimeout` ?

Reply:
yep.

Reply:
@gnufied ptal on current version.. completely removed `csiTimeout`.

Reply:
hmm, I think what @msau42 meant was 15s itself is too low of a default (and hence 10s will be even lower) and hence is it worth forcing COs/cluster-admins to specify a value rather than choosing a non-ideal default?

I see @msau42 point but I kinda like the idea of defaults, but I admit 15s is not good default for volume expansion. I wonder if we can increase the default to something like 80s which would work in most cases and if certain drivers have problems they can tune it.  

Also this PR will require fixing all our e2e jobs btw.